advanced/index.js

```diff
@@ -1 +1 @@
-var x = "Pi-hole: A black hole for Internet advertisements."
+var x = ""
```

Comment:
You can remove the file in another PR (where the source branch is on the team repo). I can adjust the tests for your.